notifications/http_test.go

```diff
@@ -16,7 +18,7 @@ import (
 // TestHTTPSink mocks out an http endpoint and notifies it under a couple of
 // conditions, ensuring correct behavior.
 func TestHTTPSink(t *testing.T) {
-	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I would prefer to keep the functionality test against a non-TLS enabled server and just run a certificate test (lines 68-88) against a TLS-enabled one.